src/engine/renderer/tr_image.cpp

```diff
@@ -1758,12 +1758,21 @@ static void R_LoadImage( const char **buffer, byte **pic, int *width, int *heigh
 		}
 	}
 
-	// the file isn't there, try again without the extension
-	COM_StripExtension3( token, filename, MAX_QPATH );
-
+	// if the file isn't there, maybe the file path did not have any extension
+	// but the file has a dot in its name that was mistakenly taken as an extension
```

Comment:
This path is also used if the filename doesn't have a dot, right? So s/but/including when/ or something.